incompatibility/incomp-scanning.go

```diff
@@ -103,3 +165,44 @@ func classificationSeverity(classification IncompatibiltiyClassification) Severi
 	}
 	return severityLevel
 }
+
+// returns a hint based the given classification
+func classificationHint(classification IncompatibiltiyClassification) string {
+	rootHint := fmt.Sprintf("%s incompatibilities occur as a result of ",
+		classification.Enum().String())
+	var reason string
+	switch classification {
+	case IncompatibiltiyClassification_Security:
+		reason = "gRPC HTTP/JSON transcoding not concerned with auth information."
+	case IncompatibiltiyClassification_ParameterStyling:
+		reason = "parameter styling not representable in .proto files."
```

Comment:
Do we want these error messages to end with "in .proto files"?  I feel like that additional context for users isn't needed?  

Thoughts?

Reply:
may seem redundant but I want to make a bigger push to indicate these incompatibilities are strongly focused .proto representation